examples/dataflow-streaming-schema-handler/src/test/java/com/google/cloud/pso/transforms/JsonSchemaExistTest.java

```diff
@@ -0,0 +1,87 @@
+package com.google.cloud.pso.transforms;
```

Comment:
Missing copyright header

Reply:
Already fix from prior push, fix from the PR of "clean up pom.xml + fix year on licenses"